boards/arm/96b_nitrogen/doc/96b_nitrogen.rst

```diff
@@ -267,22 +267,25 @@ and install a more recent version of pyOCD.
 Flashing an Application to 96Boards Nitrogen
 ============================================
 
-The sample application :ref:`hello_world` is being used in this tutorial:
+This example uses the :ref:`hello_world` sample with the
+:ref:`nxp_opensda_pyocd` tools. Use the ``make flash`` build target to build
+your Zephyr application, invoke the pyOCD flash tool and program your Zephyr
+application to flash.
 
 .. code-block:: console
 
-   $<zephyr_root_path>/samples/hello_world
+   $ cd <zephyr_root_path>
+   $ . zephyr-env.sh
+   $ cd samples/hello_world/
+   $ make BOARD=96b_nitrogen
 
-To build the Zephyr kernel and application, enter:
+You can either flash the board by using the `make flash` target:
 
 .. code-block:: console
 
-   $ cd <zephyr_root_path>
-   $ source zephyr-env.sh
-   $ cd $ZEPHYR_BASE/samples/hello_world/
-   $ make BOARD=96b_nitrogen
+   $ make flash
```

Comment:
Since the default BOARD is qemu_x86, does this need to be:

     $ make BOARD=96b_nitrogen flash